### PR TITLE
feat: add HPA for hydra +  mockTlsTermination for example-idp

### PR DESCRIPTION
**Pull request by pommelinho**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by pommelinho** [comment]
Added HPA for hydra chart.This is partially related to #194 .
Additionally i added support env MOCK_TLS_TERMINATION in example-idp chart which might be useful so that exampl-idp can communicae with hydra admin api.

File: helm/charts/hydra/values.yaml [hunk 1]
```diff
@@ -163,6 +163,14 @@ deployment:
 # Configure node affinity
 affinity: {}
 
+# Configure HPA
+autoscaling:
+  enabled: true
```
Comment (aeneasr) [inline helm/charts/hydra/values.yaml:168]: Can we move this to the "deployment" top level key? Also let's disable this per default!

**Review by aeneasr — CHANGES_REQUESTED** [review]
Awesome, thank you for your contribution! This looks pretty good but I have some ideas how to improve it further :)

File: helm/charts/hydra/values.yaml (hunk 1 above)
Comment (pommelinho) [reply helm/charts/hydra/values.yaml]: sure

**Review by aeneasr — APPROVED** [review]
Awesome 🎉 

Thank you for your contribution!

**Pull request merged**